packages/pds-ember/app/styles/pds/components/definition-list/__config.scss

```diff
@@ -0,0 +1,122 @@
+///-------------------------------------------------------------------------///
+/// WARNING: Private Sass module (definitions may change at ANY time)
+///-------------------------------------------------------------------------///
+$_module: "PDS.Components.DefinitionList";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "../../core/typography/_config" as Typography;
+@use "../../theme";
+@use "../../tokens/color";
+@use "../../tokens/font";
+@use "../../tokens/scale";
+@use "../../utils/media" as Media;
+@use "../_config" as Component;
+
+// atomic vars
+$color: color.$ui-gray-900;
+$columnGap: theme.$size--lg; // 1.5rem (24px)
+$rowGap: scale.$lg--1; // 1.25rem (20px)
+// composed vars
+$gridGap: $rowGap $columnGap;
```

Comment:
Are these going to be shared more widely or do you think they'll be specific to this component?

Reply:
not sure yet, but that's why it's defined in a private module :)